README.md

```diff
@@ -189,9 +207,6 @@ device the the remote database.
 Replication is simple to get started in the common cases:
 
 ```java
-import com.cloudant.sync.replication.ReplicatorBuilder;
-import com.cloudant.sync.replication.Replicator;
```

Comment:
With the import statements it's not a valid snippet of java. If we include the import statements we should also have the class/method definition.